bucket/stevedore.json

```diff
@@ -0,0 +1,44 @@
+{
+    "version": "0.0.4",
+    "description": "Docker distribution for Windows that Just Works",
+    "homepage": "https://github.com/slonopotamus/stevedore",
+    "license": "Apache-2.0",
+    "notes": [
+        "To install Hyper-V: `Enable-WindowsOptionalFeature -Online -FeatureName Microsoft-Hyper-V -All -NoRestart`",
+        "To install Windows Containers: `Enable-WindowsOptionalFeature -Online -FeatureName Containers -All -NoRestart` as admin.",
+        "A restart is required on the first install."
+    ],
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/slonopotamus/stevedore/releases/download/0.0.4/stevedore-0.0.4-x86_64.msi",
+            "hash": "14ccc277882fdd7e59b0f7105d0be33d32b4bfb4874e804b045ba313dedc9111"
+        }
+    },
+    "uninstaller": {
+        "script": "(Get-Service stevedored) -and (sudo Remove-Service stevedored)"
+    },
+    "pre_install": "(Get-Service stevedored) -and (sudo Stop-Service stevedored) -and (sudo Remove-Service stevedored)",
+    "post_install": [
+        "(Get-LocalGroup -Name docker-users) -or (New-LocalGroup -Name docker-users -Description 'Users of Docker Desktop')",
```

Comment:
You also want to add current user to `docker-users` group. Otherwise, user won't be able to access docker service.